packages/react/src/patterns/sub-patterns/ContentGroup/ContentGroup.js

```diff
@@ -20,41 +21,47 @@ const { prefix } = settings;
  * @param {object} props props object
  * @param {*} props.children JSX Components
  * @param {string} props.heading  Heading objects
+ * @param {string} props.className optional class to be applied to the containing node
+ * @param {*} props.cta CTA component props object
  * @returns {*} JSX ContentArrayGroup component
  */
-const ContentGroup = ({ children, heading }) => {
+const ContentGroup = ({
+  children,
+  heading,
+  className: customClassName,
+  cta,
+}) => {
+  const className = classNames(`${prefix}--content-group`, {
+    [customClassName]: !!customClassName,
+  });
```

Comment:
Any reason we couldn't just pass in `customClassName` instead of a whole object? What's the value of the object?

Reply:
On that topic is there a reason we're reassigning it to another variable `customClassName`?

Reply:
if customeClassName is not passed so it will hadle, folowwed other components has the same way.